### PR TITLE
chore: fix ESLint violation and lessen rule strictness

### PR DESCRIPTION
**Pull request by jackfranklin**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -100,6 +100,11 @@ module.exports = {
                 "@typescript-eslint/semi": 2,
                 "@typescript-eslint/no-empty-function": 0,
                 "@typescript-eslint/no-use-before-define": 0,
+                // We have to use any on some types so the warning isn't valuable.
```
Comment (paullewis) [inline .eslintrc.js:103]: :(

File: test/emulation.spec.ts [hunk 2]
```diff
@@ -67,7 +67,7 @@ describe('Emulation', () => {
       function dispatchTouch() {
         let fulfill;
         const promise = new Promise((x) => (fulfill = x));
-        window.ontouchstart = function (e) {
+        window.ontouchstart = () => {
```
Comment (paullewis) [inline test/emulation.spec.ts:70]: But why though?

**Review by paullewis — APPROVED** [review]
LGTM with curiosity question

File: test/emulation.spec.ts (hunk 2 above)
Comment (jackfranklin) [reply test/emulation.spec.ts:70]: the majority of the codebase + tests use arrow functions - this one stood out to me so I just swapped it out

File: .eslintrc.js (hunk 1 above)
Comment (jackfranklin) [reply .eslintrc.js:103]: Yeah :( Long term we'll use the TS compiler strictness options to get rid of this but these warnings are adding no value.

**Pull request merged**